src/v2/schema.rs

```diff
@@ -165,6 +165,59 @@ pub struct Operation {
     pub parameters: Option<Vec<ParameterOrRef>>,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[serde(untagged)]
+pub enum PropertyDefault {
+    Integer(i32),
+    Boolean(bool),
+    String(String),
+}
+
+impl From<i32> for PropertyDefault {
+    fn from(item: i32) -> Self {
+        PropertyDefault::Integer(item)
+    }
+}
+
+impl From<bool> for PropertyDefault {
+    fn from(item: bool) -> Self {
+        PropertyDefault::Boolean(item)
+    }
+}
+
+impl From<String> for PropertyDefault {
+    fn from(item: String) -> Self {
+        PropertyDefault::String(item)
+    }
+}
+
+impl From<PropertyDefault> for i32 {
+    fn from(item: PropertyDefault) -> Self {
+        match item {
+            PropertyDefault::Integer(item) => item,
+            _ => 1,
+        }
+    }
+}
+
+impl From<PropertyDefault> for bool {
+    fn from(item: PropertyDefault) -> Self {
+        match item {
+            PropertyDefault::Boolean(item) => item,
+            _ => true,
```

Comment:
What do you think about using rusts defaults for these i.e. returning `Default::default()`? I'm also wondering if there's a better way to communicate this wasn't the type you thought it was. I.e. if you're defererning a string default as a book ect